doc/config_regression.json.md

```diff
@@ -57,3 +57,8 @@
 **label_list**: list of lists, (default=[[“america","asia", "euorpe"], ["monday", "tuesday", "wednesday", "thurday", "friday"], ["January","Feb","Mar","Apr","May","Jun","July", "Aug", "sept.","Oct","Nov","Dev”]])
 
 		A list of lists, each list is the labels for one categorical variable.
+
+**shuffle**:string, (default=”Yes”)
```

Comment:
Why wouldn't this be a `bool`?

Reply:
I initially wrote the entire thing with `bool`. But then noticed configurations used "yes/no". So made it "yes/no" for consistency